### DISCUSSION TITLE
skyline + ts 模版初始化报错

### DISCUSSION THREAD

**Discussion by oNo500** [weapp-vite/weapp-vite#14] [General]
步骤：按照操作
报错如下
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/b12bf9e9-623d-4b7b-8aaa-03c2ae33b30b">

**Comment by sonofmagic** [comment]
目前这个启动还是需要强依赖 typescript 的，所以目前可以先安装一下来避免这个报错，后续会把 typescript 设置为可选项

**Reply by oNo500** [reply]
已解决 自己装了一个 typescript
所以最终产物是 js 文件吗？
如果使用微信开发者工具的ts转js和使用vite转js有什么区别，有点疑惑 求大佬解答

**Reply by sonofmagic** [reply]
使用微信开发者工具的ts转js, 这个是走的微信开发者工具里面内置的 typescript 插件，自定义程度要差一点

使用vite 进行ts转js，自定义程度高一些，可以使用很多 vite 插件进行扩展

weapp-vite 项目存在的目的，就是尽可能给用户提供一个高自定义和可扩展的，开发体验好的开发工具，因为微信内置那一套体验太差了。
